src/layouts/AuthenticatedLayout/SidebarItem.tsx

```diff
@@ -1,15 +1,15 @@
 import { ReactNode, cloneElement } from "react";
-import { NavLink, NavLinkProps } from "react-router-dom";
+import { Link, LinkProps, useLocation, useMatch as getMatch } from "react-router-dom";
```

Comment:
This needs to be checked.

Is there a way to do this, without renaming the hook